src/Jackett.Common/Content/custom.js

```diff
@@ -1327,15 +1327,17 @@ function updateSearchResultTable(element, results) {
                         if (keyword === "" || keyword === "+" || keyword === "-")
                             return;
                         var newKeyword;
+                        var pipedKeyword = '(' + keyword.substring(1).split('|').map(k => $.fn.dataTable.util.escapeRegex(k)).join('|') + ')';
                         if (keyword.startsWith("+"))
-                            newKeyword = $.fn.dataTable.util.escapeRegex(keyword.substring(1));
+                            newKeyword = pipedKeyword;
                         else if (keyword.startsWith("-"))
-                            newKeyword = "^((?!" + $.fn.dataTable.util.escapeRegex(keyword.substring(1)) + ").)*$";
+                            newKeyword = "^((?!" + pipedKeyword + ").)*$";
                         else
                             newKeyword = $.fn.dataTable.util.escapeRegex(keyword);
```

Comment:
Sorry, What?

Reply:
Use `pipedKeyword` on L1336 so that `x265|hevc` functions the same as `+x265|hevc`.

Reply:
There's a reason I kept it that way. In the unlikely case that there is a `|` in a torrent name which needs to be assumed literally, Keeping it without the `+` helps in doing so.
 So `|` with `+` will always be used as an OR operator and other than that, it can be used literally. Thoughts?

Reply:
In that case I'd reserve `+` for that rare case, as currently the only other use for it is wanting to filter positively by `-` (i.e. `+-` to get results which contain `-`).

So only have L1336 use `pipedKeyword`, and revert L1332 and L1334.